docs/D-Modularity/functions.md

```diff
@@ -45,7 +45,7 @@ The module named `hello.c` starts executing at statement `int main(void)`, outpu
 
 We can sub-divide a programming project in different ways.  We select our modules so that each one focuses on a narrower aspect of the project.  Our objective is to define a set of modules that simplifies the complexity of the original problem. 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/modules.png)
+![modules](https://ict.senecacollege.ca//~ipc144/pages/images/modules.png)
```

Comment:
I am not sure why it doesn't look right for yours but it looks right on mine. And @a-rokay said it was fine on his too.

![image](https://user-images.githubusercontent.com/36431357/140819666-83d4f478-d9e2-4d16-91b9-ed744f36907d.png)

Reply:
Let's see what he says before I go and rearrange the whole thing because he said it was working and mine works.

Reply:
@trangntt-016 It probably looks wrong because the changes haven't been merged. If you're trying to do it by clicking the links from the GitHub readme, then that's the old version without any of the changes I made in this issue branch.

Reply:
The update works on my side.